android/sample/src/main/java/com/facebook/flipper/sample/FlipperSampleApplication.java

```diff
@@ -33,5 +37,126 @@ public void onCreate() {
         .edit()
         .putInt("SomeKey", 1337)
         .apply();
+
+    Database1Helper db1Helper = new Database1Helper(this);
+    Database2Helper db2Helper = new Database2Helper(this);
+
+    DatabaseUtils.queryNumEntries(db1Helper.getReadableDatabase(), "db1_first_table", null, null);
+    DatabaseUtils.queryNumEntries(db2Helper.getReadableDatabase(), "db2_first_table", null, null);
+  }
+
+  public static class Database1Helper extends SQLiteOpenHelper {
+
+    // If you change the database schema, you must increment the database version.
+    public static final int DATABASE_VERSION = 4;
+    private static final String SQL_CREATE_FIRST_TABLE =
+        "CREATE TABLE " + "db1_first_table" + " (" +
```

Comment:
nit: Any reason for concatenating in the same line here?

Reply:
Easier to read SQL code formatted this way